Source/ConvolutionFunctions/arm_convolve_get_buffer_sizes_s4.c

```diff
@@ -64,6 +116,9 @@ int32_t arm_convolve_wrapper_s4_get_buffer_size(const cmsis_nn_conv_params *conv
                                                 const cmsis_nn_dims *filter_dims,
                                                 const cmsis_nn_dims *output_dims)
 {
+#if defined(ARM_MATH_MVEI)
+    return arm_convolve_wrapper_s8_get_buffer_size_mve(conv_params, input_dims, filter_dims, output_dims);
```

Comment:
Is this a typo? s8 instead of s4?

Reply:
No it is actually a reuse since they need the same size